massgov/mayflower-patternlab
# DP-7307 & DP-7483 organization image links

## Description

Create new Image Link molecule with block variant and Image Link List organism for use on the service page 'offered by' section within the sidebar, or as a stacked row item on the Organization Elected Official page.

## Related Issue / Ticket

- [DP-7307](https://jira.state.ma.us/browse/DP-7307)
- [DP-7483](https://jira.state.ma.us/browse/DP-7483)

## Steps to Test


1. View invision file https://projects.invisionapp.com/share/VYF2N5JX2#/screens/270254806
2. Confirm that the service page 'Offered By' section looks as expected: https://mayflower.digital.mass.gov/?p=pages-service 
3. View invision file https://projects.invisionapp.com/share/VYF2N5JX2#/screens
4. Confirm that the organization elected official page 'Our Organizations' section looks as expected

## Screenshots
You can see these two organisms in action here:
http://www.isaacchansky.com/mayflower/?p=pages-organization-elected-official
and here:
http://www.isaacchansky.com/mayflower/?p=pages-service

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* new `molecules/image-link`
* new `organisms/image-link-list`
* change `templates/content-types/services` to use image-link-list instead of link-list
* add to `pages/organization-elected-official` to include image-link-list for 'our organizations' section

#### @TODO


* 

#### Today I learned...
